src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate.UnitTests/TimeDateResultTests.cs

```diff
@@ -26,11 +26,16 @@ public void Setup()
             CultureInfo.CurrentUICulture = new CultureInfo("en-us");
         }
 
+        private DateTime GetDateTimeForTest()
+        {
+            return new DateTime(2022, 03, 02, 22, 30, 45);
```

Comment:
I converted all the tests hours from 10AM to 10PM, since I think it is better to test with hour whose formatting changes depending on 24h/12h AM-PM formatting preference.

Reply:
Nice catch.